src/sql/parts/modelComponents/card.component.ts

```diff
@@ -46,6 +46,34 @@ export default class CardComponent extends ComponentBase implements IComponent,
 		this.baseDestroy();
 	}
 
+	private _defaultBorderColor = 'rgb(214, 214, 214)';
+	private _hasFocus: boolean;
+
+	public onCardClick() {
+		if (this.selectable) {
+			this.selected = !this.selected;
+			this._changeRef.detectChanges();
+			this._onEventEmitter.fire({
+				eventType: ComponentEventType.onDidClick,
+				args: this.selected
+			});
+		}
+	}
+
+	public getBorderColor() {
+		if (this.selectable && this.selected || this._hasFocus) {
+			return 'Blue';
```

Comment:
This probably looks good on light and dark themes, does it also look good on high contrast? We may want to make it customizable at some point too though doesn't have to happen now

Reply:
I added color for dark and light in the css but still have to get the right color and change them later